src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.h

```diff
@@ -1859,8 +1859,14 @@ namespace UiaOperationAbstraction
         // to remote mode as soon as the remote scope begins, to avoid conditional conversion inside
         // a condition or loop. To abstract this from client code and provide symmetry with BindResult,
         // we provide a BindInput method to do this.
+        template<class... Args>
+        void BindInput(Args&... args)

```

Comment:
Technically speaking, this version allows you to provide no arguments at all to bind. Not a big deal (that would be a no-op), but maybe something to adjust?